build.gradle

```diff
@@ -38,6 +38,8 @@ dependencies {
 	implementation group: 'io.springfox', name: 'springfox-swagger-ui', version: '3.0.0'
 	implementation group: 'io.springfox', name: 'springfox-boot-starter', version: '3.0.0'
 
+	compileOnly 'org.projectlombok:lombok:1.18.20'
+	annotationProcessor 'org.projectlombok:lombok:1.18.20'
```

Comment:
Sir, if you have installed lombok why didn't you refactor the already existing classes which are still using getters and setters?(CampaignEntity, Campaign)

Reply:
I know about this problem, but I told the other guys to change it earlier.

Reply:
fixed